client/src/components/Draftail/sources/ModalWorkflowSource.js

```diff
@@ -35,7 +84,7 @@ export const getChooserConfig = (entityType, entity) => {
       // This does not initialise the modal with the currently selected text.
       // This will need to be implemented in the future.
```

Comment:
Please can you remove this comment, now that it's no longer true? :-)

Reply:
hmm I did start the commit, realised this and ganged the file before saving, I thought I had done this, will sort though. Thanks for reviewing 👍